src/testUtils.js

```diff
@@ -152,39 +152,30 @@ export async function destroyReport(report) {
     },
   });
 
-  try {
-    User.destroy({
+  const destroys = dbReport.activityRecipients.map(async (recipient) => {
```

Comment:
Nothing is actually returned from the map, so I'd assume `destroys` is always going to be empty. With `await`s being used I would think using `for (const recipient of dbReport.activityRecipients) {` instead of the  map would be clearer (and removing `the `await Promise.all(destroys)` below).

Reply:
Its needed, as the content of the mapped function is async and all the calls within are awaits. Changing to a for of loop makes awaits much less clear.

Reply:
Just had a chat with Garrett, using map here seems good